src/js/components/CheckBoxGroup/__tests__/CheckBoxGroup-test.js

```diff
@@ -56,4 +71,89 @@ describe('CheckBoxGroup', () => {
     const tree = component.toJSON();
     expect(tree).toMatchSnapshot();
   });
+
+  test('onChange', () => {
+    const onChange = jest.fn();
+    const { container, getByText } = render(
+      <Grommet>
+        <CheckBoxGroup
+          options={[
+            { label: 'first-label', value: 'First' },
+            { label: 'second-label', value: 'Second' },
+          ]}
+          onChange={onChange}
```

Comment:
There is no assertion on onChange, why is it needed?

Reply:
I added an assertion for onChange `expect(onChange).toBeCalledTimes(1);` at line 89